### PR TITLE
Enable automatic changelog generation in release workflows

### PR DESCRIPTION
**Pull request by disrupted**
Followup from #141

### CODE REVIEW

File: .github/workflows/bump-version-release.yaml [hunk 1]
```diff
@@ -11,11 +11,12 @@ on:
       changelog:
         description: "Create changelog for release."
         required: false
-        default: false
+        default: true
         type: boolean
       changelog-config:
         description: "Changelog config path."
         required: false
+        default: "$GITHUB_ACTION_PATH/changelog-config.json"
```
Comment (MichaelKora) [inline .github/workflows/bump-version-release.yaml:19]: am not sure if that'd work like this. i think in order for this to work we need to add the config file in the `bump-version-release` action dir..

File: .github/workflows/python-poetry-release.yaml [hunk 2]
```diff
@@ -25,11 +25,12 @@ on:
       changelog:
         description: "Create changelog for release."
         required: false
-        default: false
+        default: true
         type: boolean
       changelog-config:
         description: "Changelog config path."
         required: false
+        default: "$GITHUB_ACTION_PATH/changelog-config.json"
```
Comment (MichaelKora) [inline .github/workflows/python-poetry-release.yaml:33]: same as above

File: .github/workflows/bump-version-release.yaml (hunk 1 above)
Comment (disrupted) [reply .github/workflows/bump-version-release.yaml:19]: you're right. after reading the [docs on GitHub variables](https://docs.github.com/en/actions/learn-github-actions/variables) I believe `$GITHUB_ACTION_PATH` actually doesn't work at all outside of a composite action... :/

Comment (disrupted) [reply .github/workflows/bump-version-release.yaml]: I've implemented a workaround

File: actions/changelog-generate/action.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ inputs:
   config:
     description: Path to the changelog config JSON file
     required: false
-    default: "$GITHUB_ACTION_PATH/changelog-config.json"
+    default: "default"
```
Comment (yannick-roeder) [inline actions/changelog-generate/action.yaml:24]: Why not use an empty string as the default here? That would also solve the issue with having to set defaults in every workflow that uses the action

**Review by yannick-roeder — CHANGES_REQUESTED** [review]

File: actions/changelog-generate/action.yaml (hunk 3 above)
Comment (disrupted) [reply actions/changelog-generate/action.yaml:24]: good plan

Comment (disrupted) [reply actions/changelog-generate/action.yaml]: changed

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**